src/components/Inventory/view/InventorySeedlingsTable.tsx

```diff
@@ -122,6 +145,23 @@ export default function InventorySeedslingsTable(props: InventorySeedslingsTable
     return;
   };
 
+  const getFilteredNurseryName = (facilityId: number) => {
```

Comment:
We have this same function and next one here and in InventoryTable. Maybe we can export it from there? Or create a util?

Reply:
what i can do is extract out the input+filter+pills as a separate component and re-use in both tables.
i can do that later as a cleanup. created a jira https://terraformation.atlassian.net/browse/SW-2033